Author: Daniel Baumann <daniel.baumann@progress-technologies.net>
Description: Update manpage to match /sbin/btrfs (Closes: #638778, #642302).

diff -Naurp btrfs-tools.orig/man/btrfs.8.in btrfs-tools/man/btrfs.8.in
--- btrfs-tools.orig/man/btrfs.8.in	2012-06-13 15:16:21.238942325 +0200
+++ btrfs-tools/man/btrfs.8.in	2012-06-13 15:24:32.970221535 +0200
@@ -17,6 +17,8 @@ btrfs \- control a btrfs filesystem
 .PP
 \fBbtrfs\fP \fBsubvolume get-default\fP\fI <path>\fP
 .PP
+\fBbtrfs\fP \fBfilesystem df\fP\fI <path> \fP
+.PP
 \fBbtrfs\fP \fBfilesystem sync\fP\fI <path> \fP
 .PP
 \fBbtrfs\fP \fBfilesystem resize\fP\fI [+/\-]<size>[gkm]|max <filesystem>\fP
@@ -31,9 +33,9 @@ btrfs \- control a btrfs filesystem
 .PP
 \fBbtrfs\fP \fBfilesystem defragment\fP\fI <file>|<dir> [<file>|<dir>...]\fP
 .PP
-\fBbtrfs\fP \fBdevice scan\fP\fI [--all-devices|<device> [<device>...]]\fP
+\fBbtrfs\fP \fBfilesystem show\fP\fI [--all-devices|<uuid>|<label>]\fP
 .PP
-\fBbtrfs\fP \fBdevice show\fP\fI [--all-devices|<uuid>|<label>]\fP
+\fBbtrfs\fP \fBdevice scan\fP\fI [--all-devices|<device> [<device>...]]\fP
 .PP
 \fBbtrfs\fP \fBdevice add\fP\fI <device> [<device>...] <path> \fP
 .PP
@@ -132,6 +134,10 @@ Get the default subvolume of the filesys
 is similar to \fBsubvolume list\fR command.
 .TP
 
+\fBfilesystem df\fR\fI <path>\fR
+Resize a filesystem identified by \fI<path>\fR.
+.TP
+
 \fBfilesystem defragment\fP -c[zlib|lzo] [-l \fIlen\fR] [-s \fIstart\fR] [-t \fIsize\fR] -[vf] <\fIfile\fR>|<\fIdir\fR> [<\fIfile\fR>|<\fIdir\fR>...]
 
 Defragment file data and/or directory metadata. To defragment all files in a
@@ -209,7 +215,7 @@ If \fB--all-devices\fP is passed, all th
 otherwise the devices list is extracted from the /proc/partitions file.
 .TP
 
-\fBdevice balance\fR \fI<path>\fR
+\fBfilesystem balance\fR \fI<path>\fR
 Balance the chunks of the filesystem identified by \fI<path>\fR
 across the devices.
 .TP
